### ISSUE TITLE
Failed to extract native libraries when install patched apk

### ISSUE THREAD

**Issue by 327135569** [327135569/MMPatch#1]
may be should add app attribute `extractNativeLibs` ?

**Comment by 327135569** [comment]
ok, add extractNativeLibs true and solve install error. but got runtime err
```
     Caused by: java.io.FileNotFoundException: res/x3.xml
        at android.content.res.AssetManager.nativeOpenXmlAsset(Native Method)
        at android.content.res.AssetManager.openXmlBlockAsset(AssetManager.java:1050)
        at android.content.res.ResourcesImpl.loadXmlResourceParser(ResourcesImpl.java:1266)
        at android.content.res.Resources.loadXmlResourceParser(Resources.java:2295) 
        at android.content.res.Resources.getLayout(Resources.java:1187) 
        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:1211) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:1119) 
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1084) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:682) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:534) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:481) 
```
no idea

**Comment by 327135569** [comment]
fixed at fe0bb0f732c0979d8f999873468f73846c637ced
